e2e/cypress/support/renkulab-fixtures/projects.ts

```diff
@@ -41,44 +41,55 @@ function Projects<T extends FixturesConstructor>(Parent: T) {
       return this;
     }
 
-    interceptMigrationCheck(name, fixture) {
+    project(useFixture = true, path = "", name = "getProject", result = "projects/project.json") {
+      const fixture = useFixture ? { fixture: result } : undefined;
+      cy.intercept(
+        `/ui-server/api/projects/${encodeURIComponent(path)}?statistics=true`,
+        fixture
+      ).as(name);
+      return this;
+    }
+
+    interceptMigrationCheck(name, fixture, params) {
       const coreUrl = "/ui-server/api/renku/cache.migrations_check";
-      const params =
-        "git_url=https%3A%2F%2Fdev.renku.ch%2Fgitlab%2Fe2e%2Flocal-test-project.git&branch=master";
       cy.intercept(`${coreUrl}?${params}`, {
         fixture: fixture
       }).as(name);
       return this;
     }
 
-    projectMigrationUpToDate(name = "getMigration") {
+    projectMigrationUpToDate(params, name = "getMigration") {
```

Comment:
If we do this a little differently, it would be easier to override the function parameters.

```suggestion
    projectMigrationUpToDate(params={repoUrl: null, name: "getMigration"}) {
      const {
        repoUrl,
        name
      } = params;
```